app/views/submissions/_form.html.erb

```diff
@@ -57,3 +57,13 @@
     <%= f.button 'Súbory chcem len stiahnuť', class: 'govuk-button govuk-link', name: :skip_subscribe %>
   </p>
 <% end %>
+
+<script type="text/javascript" charset="utf-8">
+  document.querySelector('[name="commit"]').addEventListener('click', () => {
+    plausible('emailMeSubmission');
+  })
+
+  document.querySelector('[name="skip_subscribe"]').addEventListener('click', () => {
+    plausible('downloadSubmission');
+  })
```

Comment:
Toto presunme niekde do nejakeho analytics.js suboru co importneme. Este by bolo dobre pri tom emailme a download vediet co je to za typ submission. Lebo ich mame uz teraz viac (priznanie digital + orsr) a budu dalsie.